### ISSUE TITLE
generating ARRAY of ENUM doesn't work

### ISSUE THREAD

**Issue by malkiz-rapid** [sequelize/sequelize-auto#463]
I'm using postgres.
sql:
```sql
CREATE TYPE status_enum AS ENUM ('ACTIVE', 'DELETED');
CREATE TABLE item (
  "status" status_enum ARRAY
);
```

output of generating the model:
```javascript
const Sequelize = require('sequelize');
module.exports = function(sequelize, DataTypes) {
   return sequelize.define('item', {
     status: {
       type: DataTypes.ARRAY(null), // The Enum is not defined
       allowNull: true,
       field: 'status'
     }
    }, {
     sequelize,
     tableName: 'item'
    });
};
```

**Comment by steveschmitt** [comment]
Good catch.  What should the correct model code look like?

**Comment by malkiz-rapid** [comment]
the correct model should look like this:

```javascript
const Sequelize = require('sequelize');
module.exports = function(sequelize, DataTypes) {
   return sequelize.define('item', {
     status: {
       type: DataTypes.ARRAY(DataTypes.ENUM('ACTIVE', 'DELETED')),
       allowNull: true,
       field: 'status'
     }
    }, {
     sequelize,
     tableName: 'item'
    });
};
```

**Comment by steveschmitt** [comment]
Fixed in 0.7.6
